sally.yaml

```diff
@@ -5,6 +5,8 @@ url: go.uber.org
 packages:
     atomic:
         repo: github.com/uber-go/atomic
+    cadence:
```

Comment:
that is good point. i thought about that and i am assuming server import is not going to be imported widely(may be few) and can take a longer names(like cadence-server), where as client lib import going to  be used widely and would save few characters on import path from (cadence-client to cadence). Let me know if you don't see this pattern for client server(cadence, cadence-sever Vs cadence-client, cadence)